tests/drivers/clock_control/nrf_lf_clock_start/testcase.yaml

```diff
@@ -0,0 +1,75 @@
+tests:
+  drivers.clock.nrf_lf_clock_start_xtal_stable:
+    tags: drivers
+    platform_allow: nrf51dk_nrf51422 nrf52dk_nrf52832 nrf52840dk_nrf52840
```

Comment:
maybe put this in a common section on top.

Reply:
but the real question, can we use some filter instead of listing platforms individually?